deploy/openshift/templates/che-server-template.yaml

```diff
@@ -161,6 +161,12 @@ objects:
                 key: ca.crt
                 name: self-signed-certificate
                 optional: true
+          - name: CHE_CUSTOM_PUBLIC_CERT
```

Comment:
It's just one more SELF_SIGNED_CERT that is named as CUSTOM_PUBLIC_CERT...

I wonder if it's possible to implement more general solution which would add any number of self-signed certs which are needed.
It could be done if we mount config map values as files into a directory, and then entrypoint.sh would propagate everything from that folder to Java Trust store.

Reply:
Yes, I am going to rework it to a config map volume mount